ExampleApp/src/main/java/co/infinum/povexampleapp/GoogleInAppUpdatesExample.java

```diff
@@ -78,19 +82,18 @@ public void onRequiresUI() {
         Toast.makeText(this, "Requires UI!", Toast.LENGTH_SHORT).show();
     }
 
-    /**
-     * This method is called when Google updater finished downloading an update. It only works for Flexible updates,
-     * because on Immediate the update is installed automatically, where as on Flexible you have choice to install update whenever you
-     * or your user wants.
-     *
-     * @param handler it's used to complete update without directly sending a of Google app updater via method
-     */
+    @Override
+    public void onWrongVersion() {
```

Comment:
When is this called?

Reply:
At first, this was implemented in case something went terribly wrong with the version codes on Prince and Google play. But now when I went to check the cases in which this is invoked I think this case theoretically isn't possible but logically it is. For example there is a case where Prince has higher version than app, and that the app has higher version than Google, but in reality that case is not possible because Google cant have lower version than the app ( maybe only when we are testing things ). I left that in because I thought that it wont hurt to have that additional case, in case something horribly went wrong. What do you think? Should I remove it or leave it?

Reply:
Case when Google has lower version than app should trigger NoUpdate or UpdateNotAvailable (as we spoke before) depending on Prince's version. Right?

Reply:
Yeah, you are absolutely right. I completely forgot about that. I will make changes according to that. All cases where app version is higher than google version are not feasible.